docs/docs/FieldSet.mdx

```diff
@@ -0,0 +1,26 @@
+import { Story } from "../src/components/Story";
+
+# FieldSet
+
+> The accordion component allows the user to show and hide sections of related content on a page.
+>
+> \- [MUI Docs](https://mui.com/material-ui/react-accordion/)
+
+The Comet Admin `FieldSet` extends [Material UI's `Accordion`](https://mui.com/material-ui/react-accordion) by adding custom style and elements such as supportText, endAdornment, and options like collapsible and initialCollapsed.
+
+## Props
+
+| Name              | Type   | Default   | Description                                                                      |
+| :---------------- | :----- | :----- | :------------------------------------------------------------------------------- |
+| title          | ReactNode |  | Title of the FieldSet |
```

Comment:
Please format the table so that all rows have the same length (Prettier should do this automatically?)